makefile

```diff
@@ -96,12 +96,13 @@ $(GPODDER_SERVICE_FILE): $(GPODDER_SERVICE_FILE_IN)
 	intltool-extract --quiet --type=gettext/ini $<
 
 build: messages $(GPODDER_SERVICE_FILE) $(DESKTOP_FILES)
+	$(PYTHON) -m build --wheel --no-isolation
```

Comment:
I assume this builds a wheel, could it have its own rule? That would allow the build rule to be called without needing the build dependency, because Gentoo does soemthing else to create a wheel.

``` patch
 build: messages $(GPODDER_SERVICE_FILE) $(DESKTOP_FILES)
+
+buildwheel: build
    $(PYTHON) -m build --wheel --no-isolation
 
-install: build
+install: buildwheel
... 
-install-win: build
+install-win: buildwheel
```